<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sampler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="databaseView"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionMeasurement"/>
     <addaction name="actionSample"/>
     <addaction name="actionProcess"/>
     <addaction name="actionProject"/>
     <addaction name="actionNewDatabase"/>
    </widget>
    <addaction name="actionOpen_Database"/>
    <addaction name="separator"/>
    <addaction name="menuNew"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionCreate_Database">
   <property name="text">
    <string>Create Database</string>
   </property>
  </action>
  <action name="actionMeasurement">
   <property name="text">
    <string>Measurement</string>
   </property>
  </action>
  <action name="actionSample">
   <property name="text">
    <string>Sample</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="text">
    <string>Process Step</string>
   </property>
  </action>
  <action name="actionProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionNewDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
